dist/preview release/what's new.md

```diff
@@ -155,6 +155,7 @@
 - Set default tangents for new keys on ACE and fix their behavior ([carolhmj](https://github.com/carolhmj))
 - Let the playhead move past the last animation key on ACE ([carolhmj](https://github.com/carolhmj))
 - Improve zooming on ACE ([carolhmj](https://github.com/carolhmj))
+- Improve framing on ACE ([carolhmj](https://github.com/carolhmj))
```

Comment:
We should not mention ACE updates as ACE is new. The only line about ACE should be the one in the top section ;)

Reply:
Removed the line!

Reply:
All of them actually ;)

Reply:
@carolhmj - want to take care of this in this PR, or should I merge this and you'll submit a new one later?